### DISCUSSION TITLE
Otter generators evolution

### DISCUSSION THREAD

**Discussion by mrednic-1A** [AmadeusITGroup/otter#2613] [Ideas]
# Fixes

1. #2593 
2. https://github.com/AmadeusITGroup/otter/issues/3011
3. Error in the logs for the **command to display all the generators** (npm exec ng g [--] --help) #2755 
4. https://github.com/AmadeusITGroup/otter/issues/3012
5.  #2627

**Comment by malek-benazzouz** [comment]
Hello, I cannot edit the post for now, but I started creating some issues, I have a list with more issues, that I will create when I am back from holidays:
- https://github.com/AmadeusITGroup/otter/issues/2627
- https://github.com/AmadeusITGroup/otter/issues/2630
- https://github.com/AmadeusITGroup/otter/issues/2628
- https://github.com/AmadeusITGroup/otter/issues/2629

**Comment by malek-benazzouz** [comment]
Other issues linked to Otter generators:

- https://github.com/AmadeusITGroup/otter/issues/2663
- https://github.com/AmadeusITGroup/otter/issues/2665
- https://github.com/AmadeusITGroup/otter/issues/2666
- https://github.com/AmadeusITGroup/otter/issues/2681
- https://github.com/AmadeusITGroup/otter/issues/2620
- https://github.com/AmadeusITGroup/otter/issues/2686
- https://github.com/AmadeusITGroup/otter/issues/2687
- https://github.com/AmadeusITGroup/otter/issues/2690
- https://github.com/AmadeusITGroup/otter/issues/2691
- https://github.com/AmadeusITGroup/otter/issues/2692
- https://github.com/AmadeusITGroup/otter/issues/2714
- https://github.com/AmadeusITGroup/otter/issues/2715
- https://github.com/AmadeusITGroup/otter/issues/2718

**Comment by mrednic-1A** [comment]
This page is closed, please use the `generators` label to have an overview about the issues related to generators
